From a7a89294919144e58ae59d03f809ca3f8553ee77 Mon Sep 17 00:00:00 2001
From: Roland McGrath <roland@hack.frob.com>
Date: Fri, 2 Sep 2016 16:58:42 -0700
Subject: [PATCH 14] NaCl: Fix compile error for __dup after
 libc_hidden_proto addition.

	* sysdeps/nacl/dup.c: Add libc_hidden_def.

(cherry picked from commit 6b75ba1388bff6a81bad410d7318d385a043b3cb)
---
 ChangeLog          | 4 ++++
 sysdeps/nacl/dup.c | 1 +
 2 files changed, 5 insertions(+)

diff --git a/ChangeLog b/ChangeLog
index 2738295..1708d50 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,5 +1,9 @@
 2016-09-02  Roland McGrath  <roland@hack.frob.com>
 
+	* sysdeps/nacl/dup.c: Add libc_hidden_def.
+
+2016-09-02  Roland McGrath  <roland@hack.frob.com>
+
 	* sysdeps/posix/wait3.c: Don't treat STAT_LOC as a union, since it's
 	not any more.
 
diff --git a/sysdeps/nacl/dup.c b/sysdeps/nacl/dup.c
index 34a7cd4..cbce3f5 100644
--- a/sysdeps/nacl/dup.c
+++ b/sysdeps/nacl/dup.c
@@ -27,4 +27,5 @@ __dup (int fd)
   int result;
   return NACL_CALL (__nacl_irt_fdio.dup (fd, &result), result);
 }
+libc_hidden_def (__dup)
 weak_alias (__dup, dup)
-- 
2.7.4.GIT


